### DISCUSSION TITLE
Mapperly as MappingConfiguration for EntityFramework

### DISCUSSION THREAD

**Discussion by pSchedlm** [riok/mapperly#476] [Q&A] [answered]
Im currently searching for the best mapping tool for my project.
One question is, if Mapperly is compatible with EntityFramework like Automapper is.

Has anyone experience in that?

Best regards

**Comment by latonz** [comment, marked as answer]
What feature of Automapper are you referencing to? Mapperly does support object to object mappings, it doesn't matter if the type to be mapped is an ef core entity. Mapperly does also support `IQueryable` projections (see the [docs](https://mapperly.riok.app/docs/configuration/queryable-projections)).

**Reply by pSchedlm** [reply]
Thanks for that -> next time searching a little bit longer in the doc's.
Exactly what I'm searching for.
